### DISCUSSION TITLE
Android App: Error on switching portrait/landscape mode

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#840] [Fixed Issues]
New app version of Mar 9th:
If I switch on my smartphone (Android 9) from landscape to portrait mode or vice versa the app is re-initialized at the playback window.
The problem applied already at the former version. See also: [https://github.com/rern/rAudio-1/discussions/800#discussioncomment-2228065](url)

Rant:
I'm not fully convinced about a need of an app in addition to the browser access.
/Rant

**Comment by R101G** [comment]
@klassikmann 
The app is needed to replace the "add to home screen" option which has been removed from Android browsers. Without it, you do not default to the full-screen display needed for smaller devices.

**Reply by klassikmann** [reply]
@R101G 
I'm using Android 9, i.e. not the newest one, and "add to home screen" is still present in Firefox and Chrome.

**Reply by R101G** [reply]
I have an updated Android 11 phone which has "add to home screen" in the menu, and an Android 11 tablet which does not. I suspect that new devices lack the option, but if you are already using it they don't remove it ... or something. The situation is very confusing, so I am glad to have the app available.

**Comment by R101G** [comment]
@klassikmann 
This seems to imply that it is now up to the web page whether or not you can save it to the home page:
https://developers.google.com/web/updates/2018/06/a2hs-updates

I agree with you that the app should retain the current page when the screen is rotated, by the way.

**Reply by klassikmann** [reply]
@rern 
Any news about an update of the app?

**Comment by Raitsa** [comment]
App updated 18.5.2022:
Fix - Reload on portrait-landscape switch.

**Reply by klassikmann** [reply]
Perfect. Problem solved.
Thank you

**Reply by cmh714** [reply]
Im curious what "App" we are referring to? The Rune Audio Control app on my Pixel 6 Pro hasnt received an update since 2/19/2018 and still works as expected. No issues portrait to landscape.....and I can Add to Home Screen as well. Just curious....

**Reply by Raitsa** [reply]
> Im curious what "App" we are referring to? The Rune Audio Control app on my Pixel 6 Pro hasnt received an update since 2/19/2018 and still works as expected. No issues portrait to landscape.....and I can Add to Home Screen as well. Just curious....

Check https://play.google.com/store/apps/developer?id=rern

**Reply by cmh714** [reply]
TY!! I was unaware rern created one...will give that a try, but like I said, the old app works just fine
